resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -294,7 +295,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
       minioUrlStr: String): Unit = {
     val (minioHost, minioPort) = getServiceHostAndPort(minioUrlStr)
     val packages = if (Utils.isHadoop3) {
-      "org.apache.hadoop:hadoop-aws:3.2.0"
+      s"org.apache.hadoop:hadoop-aws:${VersionInfo.getVersion}"
     } else {
       "com.amazonaws:aws-java-sdk:1.7.4,org.apache.hadoop:hadoop-aws:2.7.6"
```

Comment:
I am reading https://github.com/apache/spark/commit/5e74570c8f5e7dfc1ca1c53c177827c5cea57bf1 and wonder if we can replace `2.7.6` -> `VersionInfo.getVersion` too.